### PR TITLE
release prep

### PR DESCRIPTION
**Pull request by FredericBlum**
Please review this PR carefully. The dataset was not previously included, but lingered around in the lexibank-repo. If I didn't miss anything in my own review, it is a good fit, but we better make sure it doesn't have serious problems.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     ],
     "creators": [
         {
-            "name": "D. T. Tryon"
+            "name": "T. Tryon"
```
Comment (SimonGreenhill) [inline .zenodo.json]: Should be Darrell T. Tryon

**Review by SimonGreenhill — COMMENTED** [review]
Only issue is the author name, Darrell T. Tryon, otherwise all looks good.

**Review by LinguList — APPROVED** [review]

File: metadata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "conceptlist": "Tryon-1983-324",
-    "citation": "Tryon, D. T. and Hackman, B. D. (1983): Solomon Islands Languages: An internal classification. Canberra: Pacific Linguistics.",
+    "citation": "Tryon, Durrell T. and Hackman, Brian D. (1983): Solomon Islands Languages: An internal classification. Canberra: Pacific Linguistics.",
```
Comment (chrzyki) [inline metadata.json]: There name is misspelled here so the dataset needs a rebuild after the metadata change.

**Review by chrzyki — CHANGES_REQUESTED** [review]

**Comment by FredericBlum** [comment]
I have added those changes now @chrzyki

**Comment by chrzyki** [comment]
> I have added those changes now @chrzyki

In https://github.com/lexibank/tryonsolomon/pull/8/commits/721bbd080cc845d5685a50504e68d4775d47050d? AFAICT it's still 'Durrell' instead of 'Darrell' in the metadata.

**Review by chrzyki — APPROVED** [review]

**Pull request merged**